refinery/ui/package.json

```diff
@@ -18,9 +17,7 @@
     "url": "https://github.com/refinery-platform/refinery-platform/issues"
   },
   "devDependencies": {
-    "autoprefixer": "6.6.1",
-    "codecov.io": "0.1.6",
-    "cypress": "^3.1.5",
+    "cypress": "^3.3.1",
```

Comment:
Can we push it to 3.4.0 (the latest version) by any chance?
https://www.npmjs.com/package/cypress

Reply:
Happens in pull request https://github.com/refinery-platform/refinery-platform/pull/3426